test/syscalls/linux/kill.cc

```diff
@@ -103,6 +103,118 @@ TEST(KillTest, CanKillAllPIDs) {
   EXPECT_EQ(0, WEXITSTATUS(status));
 }
 
+class KillSpecificReaperTest
+    : public ::testing::TestWithParam<
+          std::tuple<std::function<int(void *)>>> {
+ public:
+  static int ReaperEntryPoint(void *args) {
+    return std::get<0>(GetParam())(args);
+  }
+};
+
+constexpr pid_t kReaperPid = 1;
+
+// Reaper process of pid namespace will ignore blockable
+// signal with SIG_DFL action, such as SIGTERM.
+static int KillCannotTermReaper(void *args) {
+  if (getpid() != kReaperPid) {
+    exit(0);
+  }
+  if (kill(kReaperPid, SIGTERM) != 0) {
+    exit(0);
+  }
+  // Make sure the reaper process still alive.
+  if (kill(kReaperPid, 0) != 0) {
+    exit(0);
+  }
+  exit(175);
```

Comment:
You are using '175' to indicate a successful exit, and `0` to indicate an error, which is backwards from usual posix convention.

Can we exit(0) on success, and exit(1) on all failures?  That seems more conventional.

Reply:
I was trying to exit with a special value 175 to make sure the check is run as I excepted.
But now this useless, cause every program exit is under control, so I  have change the return logic, exit(0) on success, exit(1) on all failures.